.github/workflows/reusable_valgrind.yml

```diff
@@ -1,3 +1,4 @@
+# Run tests with valgrind intstrumentation tools: memcheck, drd, helgrind
```

Comment:
`intstrumentation` -> `instrumentation`

Reply:
I'll fix that in the next PR; I'd like to omit waiting for the whole CI to re-run 😄